code_gen/README.md

```diff
@@ -0,0 +1,34 @@
+# AI Code Gen for Pipedream Components
+
+Generate components (currently actions only) using OpenAI GPT-4.
+
+
+### Run
+
+```
+APP=slack
+PROMPT="how to send myself a direct message?"
+python run.py --app "$APP" "$PROMPT"
+python run.py --app "$APP" "$PROMPT" --no-print # do not print code output to terminal
+```
+
+
+### Installation
+
+```
+python3 -m venv ve
```

Comment:
Have you ever worked with Poetry? It manages a virtual environment / deps in one place, and we use it for most other Python projects, so I'd recommend using that to generate a `pyproject.toml` file: https://python-poetry.org/